### DISCUSSION TITLE
TSC Meeting February 2021 - Thursday 11th Feb - 4-5pm GMT

### DISCUSSION THREAD

**Discussion by lgirdwood** [thesofproject/sof#7388] [Team Posts]
Now that v1.7 is imminent it's time to look ahead and align. Still need to add more content for each item.

**Date** Thursday 11th Feb - 4-5pm GMT
Please vote using this link: https://doodle.com/poll/cccir4i7a7eeqh93?utm_source=poll&utm_medium=link

Meeting invite now sent out using Google hangout.

**Action from last TSC.**
1) Fuzzing.
2) Outreachy
3) Pipeline and Endpoint rules.
4) Mixer and demux.
5) Static analysis
6) GSoC

**Proposed Agenda**
1) 1.7 release. 
2) 1.8 release plans.
3) Linux driver updates and plans.
4) Codec adapter updates and alignment on plans.
5) Zephyr Update.
6) Fuzzing updates.
7) Topology updates.
8) cppcheck as part of CI
9) Intel cAVS Audio FW example modules & integration tools open sourcing
10) Toolchain upgrade
11) Use of assert() in code, we should be more recoverable.

**Comment by mwasko** [comment]
I have added a topic about open sourcing Intel Audio FW example modules & integration tools that are currently being used for closed source FW 3rd party module integration. We would like to keep this open and as a part of thesofproject. The long term goal is to align the 3rd party module API and integration methodology between SOF and  Windows solution.

**Comment by dbaluta** [comment]
@lgirdwood added doodle for scheduling the meeting please distribute the link.

**Comment by cujomalainey** [comment]
Doodle timeframe starts today, shall we pick a date to avoid wasting the doodle?

**Comment by dbaluta** [comment]
@lgirdwood @cujomalainey looks like Thursday, 16:00 GMT is the best time for everyone.

![image](https://user-images.githubusercontent.com/62316/107330146-dc277700-6ab9-11eb-8376-b052d8707c86.png)

**Comment by lgirdwood** [comment]
@thesofproject/steering-committee pleas shout up if you dont have an invite, I've only sent to the folks who have replied to the doodle poll. @cwakeland fyi - no one from AMD has replied.
